broker/cloud_functions/check_cue_response/deploy.sh

```diff
@@ -37,5 +37,5 @@ else # Deploy the Cloud Functions
         --entry-point "${check_cue_entry_point}" \
         --runtime python37 \
         --trigger-topic "${check_cue_trigger_topic}" \
-        --set-env-vars TESTID="${testid}",SURVEY="${survey}",VERSIONTAG="${versiontag}"
+        --set-env-vars TESTID="${testid}",SURVEY="${survey}",VERSIONTAG="${versiontag}",ZONE="${zone}"
```

Comment:
this is what I was referring to in the previous comment

Reply:
Nice. From what I can tell, this was a bug in the script.